test/sandbox/test/capybara_dsl_test.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class CapybaraDslTest < ViewComponent::TestCase
```

Comment:
For the sake of compatibility, IMO we should have a test for every method we're delegating ❤️

Reply:
Fixed: a9f88c25.